docs/source/primitives.rst

```diff
@@ -384,6 +384,21 @@ Schema
 
 .. jsonschema:: ../../src/core/schema/components/selection.json
 
+DateTime
+------
+
+A DateTime component, providing datetime selection functionality. Three types accepted in datetime_type,
+'datetime', 'date' and 'time' which represent their namesake datetime types respectively. In the case of
+datetime and date a min/max date validator ADDNAME can be provided which will block out dates which cannot be
```

Comment:
Is `ADDNAME` here intentional?

Reply:
no! good catch, I needed to check the name on the frontend